backend/scrapers/classes/parsers/baseParser.js

```diff
@@ -219,6 +219,19 @@ class BaseParser {
     return null;
   }
 
+  // gets the course Attributes. If course attributes don't exist, return null
+  // String -> [Maybe String]
+  parseStacysMom(gotItGoingOn) {
+    let courseAttributes = gotItGoingOn.match(/(Course Attributes: \n)(.*)/i);
+    if (courseAttributes) {
+      courseAttributes = courseAttributes[2].trim().split(', ');
+      for (let i = 0; i < courseAttributes.length; i++) {
+        courseAttributes[i] = courseAttributes[i].trim();
+      }
+    }
```

Comment:
want to do a 
```
else {
return null;
}
```
?

Just so its clear what we're returning if the .match didn't match anything